src/app/dummy.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+
+import { InputVerbatimDirective } from 'jslib-angular/directives/input-verbatim.directive';
+import { SearchPipe } from 'jslib-angular/pipes/search.pipe';
+
+@NgModule({
+    imports: [],
+    declarations: [
+        InputVerbatimDirective,
+        SearchPipe,
+    ],
+})
+export class DummyModule {
```

Comment:
What's this for?

Reply:
An old dummy module I removed a while back that seems to have re-appeared 😞 . (Probably during the merge conflicts). Removed again.